src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.approved.txt

```diff
@@ -1622,6 +1622,7 @@ namespace GraphQL.Types
         public string Description { get; set; }
         public string Name { get; set; }
         public object Value { get; set; }
+        public bool Visible { get; set; }
```

Comment:
No need in such custom property `Visible`. You can use `MetadataProvider`'s api to read and write any data you want.